src/examples/test_results.yml

```diff
@@ -0,0 +1,12 @@
+description: |
+  Run tests with jest and produce and upload test reports.
```

Comment:
This technically doesn't necessarily run jest but rather whatever `test` script is configured to run.

Reply:
This seems accurate to me when the second line is considered as well? it is what the job will be doing, and I can't figure out a way to put the caveat in the first line without making it less clear what the job is doing.